docs/src/components/Example.js

```diff
@@ -33,24 +34,59 @@ export default Demo;`;
 
 const handleCodeSandbox = async ({ code, title }) => {
   const gestaltComponents = Object.keys(gestalt);
+  const additionalGestaltComponents = ['DatePicker'];
+
   const usedComponents = [
     ...new Set(
       code.match(/<((\w+))/g).map(component => component.replace('<', ''))
     ),
   ];
-  const components = gestaltComponents.filter(x => usedComponents.includes(x));
+
+  const baseComponents = gestaltComponents.filter(x =>
+    usedComponents.includes(x)
+  );
+
+  const additionalComponents = additionalGestaltComponents.filter(x =>
+    usedComponents.includes(x)
+  );
+
+  const getPackagedComponents = () => {
+    const imports = [];
+    if (baseComponents.length > 0) {
+      imports.push(
+        `import "../node_modules/gestalt/dist/gestalt.css";\nimport { ${baseComponents.join(
```

Comment:
Let's import all the CSS first (the Gestalt one should always be loaded)

Reply:
fixed

Reply:
![image](https://user-images.githubusercontent.com/10593890/84557580-58767a00-ace1-11ea-9c3d-519a378e5300.png)